tests/test_pipeline.py

```diff
@@ -4,10 +4,15 @@
 
 import tempfile
 
+from spacy.language import Language
+
 from textpipe.pipeline import Pipeline
 
 TEXT = 'Test sentence for testing'
-PIPELINE = [('Raw',), ('NWords',), ('Complexity',), ('CleanText',)]
+ents_model = Language()
+model_path = '/tmp/ents_model'
+ents_model.to_disk(model_path)
+PIPELINE = [('Raw',), ('NWords',), ('Complexity',), ('CleanText',), ('Entities', {'model_path': model_path})]
```

Comment:
I'm not sure this test should be modified. I think I'd like to see the 'happy path', the simplest pipeline test (as well).